src/libs/RegExp.js

```diff
@@ -0,0 +1,11 @@
+const TIME_STARTS_01 = /^01:\d{2} [AP]M$/;
```

Comment:
This shouldn't be its own file. We currently store all regex in the const file in `CONST.REGEX`

Reply:
👍🏻

Reply:
👍🏻